ui/src/components/Dashboard/index.jsx

```diff
@@ -16,6 +16,12 @@ const useStyles = makeStyles(theme => ({
   appbar: {
     height: APP_BAR_HEIGHT,
   },
+  staging: {
+    backgroundColor: theme.palette.info.main,
+  },
+  local: {
+    backgroundColor: theme.palette.warning.main,
```

Comment:
Let's choose darker tints since the text on the app bar is white (e.g. `palette.success.dark`)

Reply:
Sure, will update it.